packages/frontend/src/components/composer/AppPickerWrapper.tsx

```diff
@@ -52,17 +52,13 @@ export const AppPickerWrapper = ({ onAppSelected, apps, setApps }: Props) => {
           app.author = url.pathname.split('/')[1]
           app.date = moment(app.date).format('LL')
         }
-        if (apps) {
-          setApps(apps)
-        }
+        setApps(apps)
```

Comment:
This if-statement seems redundant.

Reply:
Only if we return an empty array instead of null in the catch clause in getJsonFromBase64. Which makes sense. Just to make sure apps is always an (possibly empty) array.

Reply:
Updated with null check, earlier in the code.